code/modules/cargo/console.dm

```diff
@@ -107,7 +107,7 @@
 					continue
 				temp += {"<div class="spoiler"><input type="checkbox" id='[supply_name]'>"}
 				temp += {"<table><tr><td><span class="cargo32x32 [replace_characters("[N.crate_type]",  list("[/obj]/" = "", "/" = "-"))]"></span></td>"}
-				temp += {"<td><label for='[supply_name]'><b>[supply_name]</b></label></td><td><A href='?src=\ref[src];doorder=[supply_name]'>Cost: [N.cost]</A></td></tr></table>"}		//the obj because it would get caught by the garbage
+				temp += {"<td><label for='[supply_name]'><b>[N.name]</b></label></td><td><A href='?src=\ref[src];doorder=[supply_name]'>Cost: [N.cost]</A></td></tr></table>"}		//the obj because it would get caught by the garbage
```

Comment:
> supply_packs[ckey(P.name)] = P //Convert to canonical form to avoid possible problems resulting from punctuation

<hr>

> for(var/supply_name in SSshuttle.supply_packs)

Все же `doorder` уже будет с исправленным названием

Reply:
да, всё хорошо, не обратил внимание что supply_name тут уже из валидного источника